alialiwa2005.md

```diff
@@ -1,13 +1,16 @@
 ---
 name: "@alialiwa2005"
-project: "Raspberry Web Services (RWS) | DIY Homemade AWS Cloud"
+project: "Libre Web Services (LWS) | DIY Homemade AWS Cloud" 
+(also known as "Raspberry Web Services (RWS)")
```

Comment:
```suggestion
```

Reply:
we're using this section "metadata" for each project so we'd need to keep it standard with just the project name hehe, thank you!